server/boards/app/blocks.go

```diff
@@ -408,25 +408,39 @@ func (a *App) DeleteBlockAndNotify(blockID string, modifiedBy string, disableNot
 		return nil
 	}
 
-	err = a.store.DeleteBlock(blockID, modifiedBy)
-	if err != nil {
-		return err
-	}
-
 	if block.Type == model.TypeImage {
```

Comment:
This section will also need to check for the FileInfo row, similar to Attachments. If no fileInfo row, then create path and delete.